docs/dev/assets.in/scripts.js

```diff
@@ -1,3 +1,7 @@
+import algoliasearch from 'https://cdn.jsdelivr.net/npm/algoliasearch@4/dist/algoliasearch-lite.esm.browser.js';
+const client = algoliasearch('WTEWG2I8PH', 'cdf7770e846ff4e95802411b91afbba6');
```

Comment:
I believe that Chris can hook you up with the account he set up for this.

Reply:
Yes, if you'll accept this PR, I'll get access to Chris' Algolia account, update that index, get the creds, update script.js, test, etc. Thx

Reply:
It's accepted, you can merge it